src/App.tsx

```diff
@@ -1,14 +1,42 @@
-/* eslint-disable max-len */
-import React from 'react';
+import { FC, useEffect, useMemo, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { CompletedFilter } from './types/CompletedFilter';
+import { getFilteredTodos } from './utils/getFilteredTodos';
+
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
-export const App: React.FC = () => {
+export const App: FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+
+  const [searchQuary, setSearchQuary] = useState('');
```

Comment:
I see that you are using a state variable `searchQuary` to store the search query. The name `searchQuary` seems to be a typo. It's a good practice to use correct spelling in variable names to avoid confusion in the future. I suggest renaming it to `searchQuery`.